### ISSUE TITLE
Arrow keys

### ISSUE THREAD

**Issue by bzbislawski** [alexurquhart/vue-bootstrap-typeahead#37]
Would be good to navigate through option with arrow keys. Currently to choose item, use has to use mouse...

Is it possible to do that? Any workarounds?

**Comment by fabiolima** [comment]
Waiting for #44

**Comment by sohel1001** [comment]
arrow keys not work with this component

**Comment by mattzollinhofer** [comment]
I merged a fix for this into my fork (https://github.com/mattzollinhofer/vue-typeahead-bootstrap) of this project that I'm actively maintaining. Arrow keys should be working now!

You can find it on npm here: https://www.npmjs.com/package/vue-typeahead-bootstrap

I hope this helps!

**Comment by loics2** [comment]
Did @alexurquhart stop maintaining this repo?

**Comment by mattzollinhofer** [comment]
Yeah it looks like it. I reached out a number of ways to see if I could help and didn’t get a response. #60  has a few more details too.

**Comment by lisandromc** [comment]
> Did @alexurquhart stop maintaining this repo?

He had no activity whatsoever on GitHub for the past 18 months:
https://github.com/alexurquhart?tab=overview&from=2019-12-01&to=2019-12-31

@azinamo has approved my PR but I don't see the commit in the tree.

@mattzollinhofer if your fork is working fine, it would be best to have NPM update their reference to your fork. Or, if that isn't practicable, update the README so that installation instructions won't resolve to the dead fork.

**Comment by mattzollinhofer** [comment]
Yeah, my fork is working fine. Unfortunately, I have no control over this repository or npm so I can't update this README or npm. All I can do is provide a good solution and support through issues on here. Eventually it should work out, but it will take time.
